packages/testcontainers/src/containers/NativeChainRpc.ts

```diff
@@ -0,0 +1,231 @@
+import { ExecResult } from 'testcontainers/dist/docker/types'
+import { StartedNativeChainContainer } from './NativeChainContainer'
+import fetch from 'cross-fetch'
+
+export class NativeChainRpc {
```

Comment:
We should point the old testcontainers RPC calls into this Class so we don't duplicate codes. Having just one "Rpc" helper to maintain.

```ts

export class NativeChainRpc {
  constructor (private readonly rpcUrl: string) {
  }

// ...
```

```ts
export class MasterNodeRegTestContainer extends RegTestContainer {
  private rpc = new NativeChainRpc(this)

  async generate (...): Promise<void> {
    return this.rpc.generate(...);
  }
```

Reply:
> the reason behind this suggestion — since we don't plan to deprecate the old testcontainers and we plan to support both environments, this suggestion make sure we always have both maintained and pointing to the same abstraction

Reply:
Yup, I'm working on it now.